opentracing/src/main/java/opentracing/TraceContext.java

```diff
@@ -14,53 +14,22 @@
 package opentracing;
 
 /**
- * TraceContext encpasulates the smallest amount of state needed to describe a Span's identity
- * within a larger [potentially distributed] trace. The TraceContext is not intended to encode the
- * span's operation name, timing, or log data, but merely any unique identifiers (etc) needed to
- * contextualize it within a larger trace tree.
- *
- * <p>TraceContexts are sufficient to propagate the, well, *context* of a particular trace between
- * processes.
- *
- * <p>TraceContext also support a simple string map of "trace attributes". These trace attributes
- * are special in that they are propagated *in-band*, presumably alongside application data. See the
- * documentation for {@link #setAttribute(String, String)} for more details and some important
- * caveats.
+ * Long-lived interface that knows how to create a root {@link SpanContext} and encode/decode
+ * any other.
  */
 public interface TraceContext {
```

Comment:
this is not a context object (anymore), and it's not concerned with a specific trace (given its long-lived-ness)... as such I am puzzled about this name...